activejob/test/integration/queuing_test.rb

```diff
@@ -43,6 +43,15 @@ class QueuingTest < ActiveSupport::TestCase
     end
   end
 
+  test "resque JobWrapper should have instance variable queue" do
+    skip unless adapter_is?(:resque)
+    Sidekiq::Testing.inline! do
```

Comment:
Does it needs to use sidekiq here? It should skip this test if the adapter is no resque.